templates/components/products/bulk-discount-rates.html

```diff
@@ -1,12 +1,12 @@
 {{#if bulk_discount_rates.length}}
     {{#if product}}<dt class="productView-info-name">{{lang 'products.bulk_pricing.title'}}</dt>{{/if}}
     <dd class="productView-info-value">
-        <a href="{{url}}"
-           {{#unless is_ajax }}data-reveal-id="bulkPricingModal{{#if id}}-{{id}}-{{$index}}{{/if}}"{{/unless}}>
+        <a href="{{product.url}}#bulk_pricing"
+           {{#unless is_ajax }}data-reveal-id="modal-bulk-pricing"{{/unless}}>
```

Comment:
Removed the `id` identifier because this modal is only used here. Presumably this id is here in the case of collisions. Uses the `modal-some-name` pattern used in `modal-review-form`.

Reply:
🍹 from a "minimum viable change" perspective, is this needed? Or did this just feel like good cleanup? I believe bulk discount rates _can_ be rendered on PLPs with [certain configurations of theme config](https://github.com/bigcommerce/cornerstone/blob/master/config.json#L334) in which case this might serve a purpose (collisions, as you mentioned)

Reply:
It felt like cleanup. Reverting that change until I'm absolutely 100% sure it wouldn't affect anything.

Reply:
I was wrong. `product.url` **is** necessary. It makes no difference on the product details page, but it's necessary where multiple products are shown, like a category page. If it's just `url`, the link won't actually go anywhere. It'll go to a nonexistent anchor on the same page.